### PR TITLE
Highlight our community and activities

### PR DESCRIPTION
**Pull request by Relequestual**
Highlight and link to community discussions, regular calls, and our slack server.
Resolves json-schema-org/community/issues/98
Resolves json-schema-org/community/issues/99
Resolves json-schema-org/community/issues/100

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -20,21 +20,48 @@ permalink: /
...
+
+We have our other [learning resources](/learn), including the [Understanding JSON Schema documentation](/understanding-json-schema).
 
-You can also see our other [learning resources](/learn).
+## About Our Community
 
-### Got questions?
+We have an active and growing community. All are welcome to be part of our community, help shape it, or simply observe.
+
+We want keep our community welcoming and inclusive, so please read our [JSON Schema Organizational Code of Conduct](https://github.com/json-schema-org/.github/blob/main/CODE_OF_CONDUCT.md). (This is a combination of the Contributor Covenant and IETF BCP 54.)
```
Comment (jdesrosiers) [inline index.md:56]:
```suggestion
We want to keep our community welcoming and inclusive, so please read our [JSON Schema Organizational Code of Conduct](https://github.com/json-schema-org/.github/blob/main/CODE_OF_CONDUCT.md). (This is a combination of the Contributor Covenant and IETF BCP 54.)
```

File: index.md [hunk 2]
```diff
@@ -20,21 +20,48 @@ permalink: /
...
 
 The JSON Schema team and community are here to help!
 
-At any point, feel free to join our [Slack server](/slack).
+At any point, feel free to join our Slack server. [Slack server](/slack).
```
Comment (jdesrosiers) [inline index.md:60]: Is the change here intended? The repetition is a bit awkward.

**Review by jdesrosiers — CHANGES_REQUESTED** [review]
Found a couple minor issues.

I'm personally not a fan of the "Read more" buttons. I'd rather just see all the content. If something is large enough to be a bother it's better off being on its on page.

File: index.md (hunk 1 above)
Comment (karenetheridge) [reply index.md:56]: Github's email produces an alert here:

> This file contains bidirectional Unicode text that may be interpreted or compiled differently than what appears below. To review, open the file in an editor that reveals hidden Unicode characters. [Learn more about bidirectional Unicode characters](https://github.co/hiddenchars)

File: index.md (hunk 2 above)
Comment (Relequestual) [reply index.md:60]: Gosh, no. Thank you! Let me fix that!

Comment (Relequestual) [reply index.md:60]:
```suggestion
At any point, feel free to join our [Slack server](/slack).
```

File: index.md (hunk 1 above)
Comment (Relequestual) [reply index.md]: > Github's email produces an alert here:
> 
> > This file contains bidirectional Unicode text that may be interpreted or compiled differently than what appears below. To review, open the file in an editor that reveals hidden Unicode characters. [Learn more about bidirectional Unicode characters](https://github.co/hiddenchars)

Which email?
I don't believe there are any bidirectional unicode text. I have added some unicode characters.
I don't see the same warning.
You should also see the warning when viewing the file on GitHub, and I do not.

**Comment by Relequestual** [comment]
> I'm personally not a fan of the "Read more" buttons. I'd rather just see all the content. If something is large enough to be a bother it's better off being on its on page. - @jdesrosiers 

I'm a fan, but I also agree. I'm trying to keep the changes small and incremental, so I'll log a new issue for migrating content from the home page to their own pages.
There is a challenege as to how you find those pages, as they may not fit in the menu.

File: index.md (hunk 1 above)
Comment (karenetheridge) [reply index.md]: The github notification emails (and it also shows up in the slack notifications).

I suspect it's a bug with how github is generating diffs, and it's triggering their own heuristic.

**Review by jdesrosiers — APPROVED** [review]

**Pull request merged**